src/pages/MyAdvertsPage.jsx

```diff
@@ -0,0 +1,23 @@
+import { Link } from "react-router-dom";
+
+
+const MyAdvertsPage = () => {
+    return (
+        <div>
+            <div style={{ marginTop: '50px', marginBottom: '50px', fontSize: '50px', textAlign: 'center' }}>
+                Here will be a list of my posts!
+            </div>
+            
+            <Link
+                to='/add-advert'
+                style={{ fontSize: '30px', backgroundColor: '#D7DD73', padding: '8px', color: '#FFFFFF', borderRadius: '10px' }}
```

Comment:
Please apply styles-components approach for this element

Reply:
This code only for convenience, because design for this part is apsent. 
That is way styles-components didn`t used.

Reply:
This part of code has removed.